src/components/flame-graph/Canvas.css

```diff
@@ -15,3 +15,14 @@
 .flameGraphCanvasTooltip {
   padding: 5px;
 }
+
+.flameGraphCanvasTooltip > .tooltipHeader {
+  display: grid;
+  grid-template-columns: min-content auto;
+  grid-gap: 2px 0;
+}
+
+.flameGraphCanvasTooltip .tooltipTitle {
+  white-space: nowrap;
+  overflow: visible;
```

Comment:
Not adding this will cause tooltips to become truncated with an ellipsis, which is something desirable, but it does it "too much" when displaying tooltips close to the edge of the chart. I'm adding this to retain the current behavior and defer the solution to "too long tooltips"-problem to a separate PR.